modules/virt-importing-vm-cli.adoc

```diff
@@ -169,8 +170,9 @@ EOF
 <9> If `storageMappings` are specified in both the `ResourceMapping` and the `VirtualMachineImport` CRs, the `VirtualMachineImport` CR takes precedence.
 <10> Specify the source storage domain.
 <11> Specify the target storage class.
-<12> Specify the source VM disk ID, for example, `8181ecc1-5db8-4193-9c92-3ddab3be7b05`. You can obtain the disk ID by entering `\https://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` in a web browser on the Manager machine and reviewing the VM details.
-<13> Specify the target storage class.
+<12> Specify `ReadWriteMany` or `ReadWriteOnce`.
```

Comment:
@pkliczewski  Do the access settings require more explanation in this note?

Reply:
I would add that it is volume access mode.